themes/custom/az_barrio/config/schema/az_barrio.schema.yml

```
@@ -302,7 +260,7 @@ az_barrio.settings:
       type: boolean
       label: 'Enable privacy link'
     copyright_notice:
-      type: text
+      type: string
```

Let's leave this translatable
```suggestion
      type: text
```